src/database/migrations/20220321123904-create-trip.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+export async function up(queryInterface, Sequelize) {
+    await queryInterface.createTable('trips', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+      },
+      departure: {
```

Comment:
Add userId and accomodationId in the migration as well

Reply:
He has done that in this migration src/database/migrations/20220411131632-create-trip-accomodation-relationship.js

Reply:
@iyaremye-faustin